database/models/links.db.php

```diff
@@ -0,0 +1,32 @@
+<?php
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\ManyToMany;
+use Doctrine\ORM\Mapping\Table;
+
+#[Entity, Table(name: 'links')]
+class Link
+{
+    #[Id, Column, GeneratedValue]
+    public int|null $id = null;
+
+    #[Column]
+    public string $url;
+
+    #[Column]
+    public string $button_text;
+
+    #[Column]
+    public string $description;
+
+    public function __construct($url = null, $button_text = null, $description = null)
```

Comment:
I don't know if Doctrine deals well with null values for string. What happens if you create an entity with those default values? It might result in a bug but I am not sure.

Reply:
Yeah you're right, Doctrine won't link that, changed it !